### ISSUE TITLE
green-coffee vs cucumber

### ISSUE THREAD

**Issue by mlsem** [mauriciotogneri/green-coffee#23]
I have rather a question than a remark :)

I've used green coffee in one of my previous projects, it was awesome, it fitted well to the project needs ( app project developed with Java)

Now I am facing a new project, a library android project, using Java and going to use kotlin also in the project, and i can not find any comparing documents between green-coffee and cucumber.

my question maybe will not be an exact question, is green-coffee could really replace cucumber ? 

is there any recommendations or project specifications ( technical or other)  that make us choose green-coffee over cucumber or the opposite ?   

I will be really happy if you could give me your opinion, over my questions , also i think its not only my question, maybe on light of your response, we could add to the wiki an objectify comparison between those tow libraries :)

**Comment by mauriciotogneri** [comment]
Hi @mlsem 

First, I'm glad the library is being useful to you :)

However, I don't know if I can help you here since I never used Cucumber. I tried more than 3 years ago and since it was very difficult to make it work for an Android project I created this library out of frustration. Maybe now things have gotten better but I don't really know.

Also, to be honest, I'm no longer maintaining this repository. I have moved on to other projects and I don't have time to keep improving this one.

So a point in favor for Cucumber is that at least is a well maintained project. Unfortunately I can't help you comparing both solutions.
